### PR TITLE
Supporters logos

### LINKED ISSUE #2036: Update the sponsors/foundation members
Per https://matrix.org/membership/

Silver should have a logo on the sponsors page
Gold should have a bigger logo on the sponsors page
Platinum should have a logo on the frontpage

### PR DESCRIPTION
**Pull request by thibaultamartin**
Closes #2036

### CODE REVIEW

**Review by MTRNord — CHANGES_REQUESTED** [review]
The images are in the wrong folder I think. Locally it tries http://127.0.0.1:1111/supporters/beeper.svg which is 404

**Review by MTRNord — APPROVED** [review]
Works now and seems good

**Comment by thibaultamartin** [comment]
I tried adding Gematik's logo, and it showed me that my CSS needs to be improved! On it

File: sass/_support.scss [hunk 1]
```diff
@@ -64,6 +64,48 @@
...
+                    border-radius: 16px;
+                    padding-top: 4px;
+                    padding-bottom: 4px;
+                    padding-left: 8px;
+                    padding-right: 8px;
+
+                    img {
+                        aspect-ratio: 1/1;
+                        object-fit: contain;
+                        width: 150px;
```
Comment (thibaultamartin) [inline sass/_support.scss:102]:
```suggestion
                        width: var(--logo-width);
```

And defining `--logo-width` in the `#gold-supporters` and `#silver-supporters` sections as suggested below

File: templates/support.html [hunk 2]
```diff
@@ -30,6 +31,30 @@ <h2>Join the Matrix.org Foundation</h2>
         </p>
         <a href="/membership" class="call-to-action">Become a member</a>
     </div>
+    <div id="supporters">
+        <div class="supporters-section">
```
Comment (thibaultamartin) [inline templates/support.html:35]:
```suggestion
        <div id="gold-supporters" class="supporters-section">
```

File: templates/support.html [hunk 3]
```diff
@@ -30,6 +31,30 @@ <h2>Join the Matrix.org Foundation</h2>
...
+            <div class="cards">
+                {% for supporter in supporters.gold %}
+                <a href="{{ supporter.website }}" class="supporters-card">
+                    <img src="/support/{{ supporter.logo }}" alt="{{ supporter.name }}'s logo">
+                    <span>{{ supporter.name }}</span>
+                </a>
+                {% endfor %}
+            </div>
+        </div>
+        <div class="supporters-section">
```
Comment (thibaultamartin) [inline templates/support.html:46]:
```suggestion
        <div id="silver-supporters" class="supporters-section">
```

**Review by thibaultamartin — COMMENTED** [review]
What do you think?

File: templates/support.html (hunk 3 above)
Comment (MTRNord) [reply templates/support.html]: Applied in c3118df625fdeb83ca1abdbbc26db8293f376db4

File: templates/support.html (hunk 2 above)
Comment (MTRNord) [reply templates/support.html]: Applied in c3118df625fdeb83ca1abdbbc26db8293f376db4

File: sass/_support.scss (hunk 1 above)
Comment (MTRNord) [reply sass/_support.scss]: Applied in c3118df625fdeb83ca1abdbbc26db8293f376db4

**Pull request merged**